hi-ml-histopathology/src/histopathology/utils/analysis_plot_utils.py

```diff
@@ -171,8 +172,8 @@ def _plot_crossval_roc_and_pr_curves(crossval_dfs: Dict[int, pd.DataFrame], roc_
     """
     for k, tiles_df in crossval_dfs.items():
         slides_groupby = tiles_df.groupby(ResultsKey.SLIDE_ID)
-        labels = slides_groupby[ResultsKey.TRUE_LABEL].agg(pd.Series.mode)
-        scores = slides_groupby[ResultsKey.PROB].agg(pd.Series.mode)
+        labels = slides_groupby[ResultsKey.TRUE_LABEL].first()
```

Comment:
Do we expect all the tiles in the group to have the same label & score? If not, should we explain in the docstring the decision to take the first in the group, rather than the most frequent?

Reply:
Yes, the CSV is constructed such that all tiles from the same slide have the same label and score. Mode was an arbitrary way to get this unique value, but I think it was somehow was failing in my latest environment.

Reply:
Just to make sure and make the code slightly more explicit, shall we add something something like:

```python
label_series = slides_groupby[ResultsKey.TRUE_LABEL]
assert len(label_series.unique()) == 1
labels = label_series.first()
```